next.config.js

```diff
@@ -14,6 +14,10 @@ const nextConfig = {
   sassOptions: {
     prependData: `@import "styles/variables"; @import "styles/mixins";`,
   },
+  images: {
+    loader: 'imgix',
+    path: 'https://noop/',
+  },
```

Comment:
I don't think this is right? It will break production for sure 🤔

Reply:
It fails on the Preview environment as well:

![image](https://user-images.githubusercontent.com/111746/126976851-cf050a64-fa03-40c4-aa42-d2d5b141ea0d.png)

Reply:
I've just removed and I remembered why we added it, because if we don't, then we can't generate the site, `npm run export` fails:
![Screenshot 2021-07-26 at 12 58 21](https://user-images.githubusercontent.com/39853718/126978398-9cc38d10-1296-44c1-b6e1-a473fee682a6.png)

Reply:
That's right! With Next, if we want to use the `Image` component, we can't just do `npm export`. That's why we're actually using `npm run build` and leveraging Netlify workers.

Next is a bit interesting as it automatically determines if a page needs to use Server Side Rendering or not depending on the hook you use to fetch the data: https://nextjs.org/docs/basic-features/pages#two-forms-of-pre-rendering

So, long story short: We can just use `npm run build` instead of `npm run export` and it will basically do the same as far as we use (`getStaticProps`)[https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation]

Reply:
But for the e2e tests we need the generated site.

If you take a look at the e2e workflow, first we generate the site, then we serve it and we run the tests on the generated site:
![Screenshot 2021-07-26 at 13 13 50](https://user-images.githubusercontent.com/39853718/126980105-dbd8f268-f48b-4e4c-a117-39245bda523c.png)

Reply:
Oh...! This is going to be tricky then. Could we spin un a server instead? Should be doable with `npm run start`.